jumpscale/packages/tfgrid_solutions/chats/peertube.py

```diff
@@ -149,6 +149,7 @@ def select_domain(self):
 
         self.gateway = domains[self.domain]
```

Comment:
would it affect to use self.domain here before correcting the domain?

Reply:
no, that is another thing, it first select domain, then and prefix to it 
self.doamin is like threefold.com 
self.domain after workds is user-solution.threefold.com
bad naming :)